src/test/java/roomescape/ThemeTest.java

```diff
@@ -0,0 +1,176 @@
+package roomescape;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.annotation.DirtiesContext;
+import roomescape.theme.domain.Theme;
+import roomescape.theme.error.exception.ErrorCode;
+import roomescape.theme.error.exception.ThemeException;
+import roomescape.theme.presentation.dto.ThemeRequest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+public class ThemeTest {
+
+    private static final String RENDERING_THEME_URL = "/admin/theme";
+    private static final String THEMES_API_URL = "/themes";
+    private static final String PATH_VARIABLE_SUFFIX_URL = "/1";
+    private static final String NAME = "name";
+    private static final String ID = "id";
+    private static final String THEME_NAME = "레벨2 탈출 희망";
+    private static final String DESCRIPTION = "description";
+    private static final String THEME_DESCRIPTION = "우테코 레벨2를 탈출하는 내용입니다";
+    private static final String THUMBNAIL = "thumbnail";
+    private static final String THEME_THUMNAIL_URL = "https://i.pinimg.com/236x/6e/bc/46/6ebc461a94a49f9ea3b8bbe2204145d4.jpg";
+    private static final int ONE = 1;
+    private static final String WILD_CARD = "$";
+    private static final String RIGHT_NAME = "TwentyLengthsExample";
+    private static final String RIGHT_DESCRIPTION = "RightDescription";
+    private static final String RIGHT_THUMBNAIL_URL = "https://edu.nextstep.camp/spring";
+
+    @Test
+    void 테마_관리_페이지를_랜더링한다() {
+
+        //when
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+                .when().get(RENDERING_THEME_URL)
+                .then().log().all()
+                .extract();
+
+        //then
+        Assertions.assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
```

Comment:
```suggestion
        RestAssured.given().log().all()
            .when().get(RENDERING_THEME_URL)
            .then().log().all()
            .assertThat()
            .statusCode(HttpStatus.OK.value());
```

현재 사용하고 계시는 `RestAssured` 는 꽤 기능이 풍부한 라이브러리라는 사실 알고 계신가요?
assertJ 가 거의 필요없을 정도로 이것만을 써서 통합테스트를 작성할 수 있을 정도인데요,
한 번 사용해보는 것도 괜찮을 것 같아요 :)